app/views/shared/_header.html.erb

```diff
@@ -30,7 +30,7 @@
             <ul class="dropdown-menu">
               <% Environment.all.each do |env| %>
                 <li>
-                  <%= link_to dashboard_path(env) do %>
+                  <%= link_to dashboard_path(env.name) do %>
```

Comment:
true true, but at least it's consistent :)

On Wed, Mar 11, 2015 at 9:22 AM, Shane Hender <notifications@github.com>
wrote:

> In app/views/shared/_header.html.erb
> <https://github.com/zendesk/samson/pull/288#discussion_r26227982>:
>
> > @@ -30,7 +30,7 @@
> >              <ul class="dropdown-menu">
> >                <% Environment.all.each do |env| %>
> >                  <li>
> > -                  <%= link_to dashboard_path(env) do %>
> > +                  <%= link_to dashboard_path(env.name) do %>
>
> Aye, switching to use to_param override will necessitate me changing all
> the admin pages as well for environments. But I can do that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/288/files#r26227982>.
>